test/modules/aws-rds-integration.ts

```diff
@@ -61,7 +61,7 @@ describe('RDS Integration Testing', () => {
   it('creates an RDS instance', query(`
     BEGIN;
       INSERT INTO rds (db_instance_identifier, allocated_storage, db_instance_class, master_username, master_user_password, availability_zone, engine, backup_retention_period)
-        VALUES ('${prefix}test', 20, 'db.t3.micro', 'test', 'testpass', '${availabilityZone}', 'postgres:13.4', 0);
+        VALUES ('${prefix}test', 20, 'db.t3.micro', 'test', 'testpass', '${availabilityZone}', 'postgres:14.1', 0);
```

Comment:
why 14.1 here? aren't we upgrading to 14.4 ?

Reply:
This is the test for RDS integration and RDS does not let you upgrade from 13.4 to 14.4, I think they only allow smaller gaps for engine updates, so he is creating the first RDS instance with 14.1 and updating to 14.4 in line 85.

Reply:
That's right, but it also brings up a wrinkle in us moving staging and production over to Postgres 14, which is: how to convince RDS to do it. We may have to make a snapshot and restore that snapshot into a new RDS instance, then swap the `db.iasql.com` and similar domains over, but that means we need explicit downtime to make the jump, which isn't great.